src/Features/CSharp/Portable/InitializeParameter/CSharpAddParameterCheckCodeRefactoringProvider.cs

```diff
@@ -49,5 +62,10 @@ protected override bool CanOffer(SyntaxNode body)
 
             return true;
         }
+        protected override SyntaxNode GetParameterNodeAtIndex(int ordinal, SyntaxNode functionDecleration)
```

Comment:
nit: probably should put `functionDecleration` before `ordinal`, since we are using `ordinal` to index into `functionDecleration`